latest/docs-ref-autogen/ext/subscription/account.yml

```diff
@@ -25,10 +25,11 @@ directCommands:
     parameterValueGroup: MS-AZR-0017P, MS-AZR-0148P
     summary: The offer type of the subscription. For example, MS-AZR-0017P (EnterpriseAgreement) and MS-AZR-0148P (EnterpriseAgreement devTest) are available.
     description: ''
-  optionalParameters:
-  - name: --display-name
+  - isRequired: true
```

Comment:
The doc is auto-generated and the change will be overwritten in the next release. 

If `display_name` is required, it shouldn't have default value `None`:

https://github.com/Azure/azure-cli-extensions/blob/8cca699619480fc0dc4d6283e60e87b75bb191d9/src/subscription/azext_subscription/custom.py#L78

```py
def cli_subscription_create(cmd, client, enrollment_account_name, offer_type,
                            display_name=None, owner_object_id="", owner_spn="",
                            owner_upn=""):
```
\+ @wilcobmsft to confirm.

Reply:
If that is the case I don't have a problem submitting a PR to update that.